### DISCUSSION TITLE
Moving to GPO based control

### DISCUSSION THREAD

**Discussion by 25garfield** [Romanitho/Winget-AutoUpdate#776] [Q&A]
We've been installing it via ssm and command line with a few settings and then creating our own centrally managed scheduled tasks, we want to now move to GPO controlled, I've got winget auto update installed on the machine already and also a gpo that mimics the local settings, should the gpo control just take over or do I need to do something else? How do I even know if winget auto update is even getting the GPO settings?

**Comment by 25garfield** [comment]
I think I've worked it out, within the policy script are list registry entries that show where the settings are that should be used and those values are being populated,

Computer\HKEY_LOCAL_MACHINE\SOFTWARE\Policies\Romanitho\Winget-AutoUpdate
Computer\HKEY_LOCAL_MACHINE\SOFTWARE\Romanitho\Winget-AutoUpdate

The only thing is I can't see where its setting the time to run the scripts etc. by default its runs at login and at 6am but if I change those settings via the gpo those changes get to the registry keys but the default scheduled task don't change?

**Reply by KnifMelti** [reply]
https://github.com/Romanitho/Winget-AutoUpdate/discussions/767#discussioncomment-11303266

**Comment by 25garfield** [comment]
Thanks, it just looks like its taken a few restarts and runs of the policies to appy, but its all good again now

Thanks
